build_release.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+nuget restore
+# Options Debug|Release
+CONFIGURATION=Release
+msbuild ResxTranslator/ResxTranslator.csproj /property:Configuration=$CONFIGURATION
+rm -rf $CONFIGURATION
+
+mkdir -p $CONFIGURATION/lib
+mkdir -p $CONFIGURATION/bin
+mv ResxTranslator/bin/$CONFIGURATION $CONFIGURATION/lib/ResxTranslator
+echo "#!/bin/sh
+SCRIPT_DIR=\$( cd -- \"\$( dirname -- \"\${BASH_SOURCE[0]}\" )\" &> /dev/null && pwd )
+exec mono \$SCRIPT_DIR/../lib/ResxTranslator/ResxTranslator.exe \"\$@\"" >> $CONFIGURATION/bin/resx_translator
```

Comment:
Wouldn't it be better to target .NET 6 ? anyways, that's work for later on.

Reply:
I'm open to exploring (but _admittedly_ less familiar).

In the future I might another PR together